liquibase-core/src/main/java/liquibase/parser/core/java/JavaChangeLogParser.java

```diff
@@ -0,0 +1,61 @@
+package liquibase.parser.core.java;
```

Comment:
This is the main new class I added. It's job is to take the anotated java class and convert it into a DatabaseChangeLog object. 

I tried to keep it bare-bones but showing how we can take an annotated changelog class and run it through the normal Parser logic. By doing it this way, we should be able to keep the java support purely in the changelog parsing logic and leave all the other bootstrapping/cli/spring/ant/etc code unchanged rather than having to re-implement a lot of like your LiquibaseRunner class was starting to have to do.